labs/RivePlayer/samples/RivePlayer.Samples/RivePlayer.md

```diff
@@ -17,6 +17,181 @@ subcategory: Input
 
 # RivePlayer
 
-This is a high level control for playing a [Rive state machine](https://rive.app/features#state-machine) or animation. A state machine's specific [inputs](https://help.rive.app/editor/state-machine#inputs) can be declared in XAML as nested properties, and controlled interactively via code or data binding.
+![Rive hero image](https://rive-app.notion.site/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2Fff44ed5f-1eea-4154-81ef-84547e61c3fd%2Frive_notion.png?table=block&id=f198cab2-c0bc-4ce8-970c-42220379bcf3&spaceId=9c949665-9ad9-445f-b9c4-5ee204f8b60c&width=2000&userId=&cache=v2)
 
-> [!Sample RivePlayerCustomSample]
+A high-level runtime for the [Windows Community Toolkit](https://docs.microsoft.com/windows/communitytoolkit/) to use [Rive](https://rive.app) in Universal Windows Platform (UWP) applications.
+
+This library allows for control over Rive files with a high-level API for driving [state machines](https://help.rive.app/editor/state-machine) and animations.
+
+
+## Getting Started
+
+To add the Rive API to your applications, include the namespace in your XAML:
+
+```xml
+<Page 
+    ...
+    xmlns:Rive="using:CommunityToolkit.Labs.WinUI.Rive"
+>
+```
+
+Note: This package is a wrapper around the [RiveSharp](https://github.com/rive-app/rive-sharp) library; a low-level API for Rive with C# that is used to build the render loop.
+
+## Classes
+
+### `<Rive:RivePlayer>`
+
+High-level control class for a Rive instance to display on a canvas. It is declared in XAML files, and controlled via code or data-binding.
+
+- Declared in XAML files
+- Controlled via code or data binding
+
+### Props
+
+- `Width` - (double) Width of the canvas
+- `Height` - (double) Height of the canvas
+- `Source` - (string) URI to the `.riv` file to load into the app. Supported schemes are HTTP, HTTPS, and `ms-appx`
```

Comment:
Why the different style fo HTTP and ms-appx? Is HTTP a variable somewhere?

Reply:
Nope, I can change them all to "code"-flavored markdown

Reply:
Last question. Why is one capitalized and the other not?

Reply:
no particular reason, we can change it to lowercase